### PR TITLE
106: do not pass 'helperText' down to 'TextInput'

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

**Comment by wtrocki** [comment]
@ljmotta I hope that this change would not affect your implementation in any way. We are only adding new hints to pages.

File: src/TextField.tsx [hunk 1]
```diff
@@ -170,6 +174,10 @@ const Text = (props: TextFieldProps) => {
         value={props.value ?? ''}
         {...filterDOMProps(props)}
       />
+      <HelperText>
+        <HelperTextItem>{props.helperText}</HelperTextItem>
+      </HelperText>
```
Comment (wtrocki) [inline src/TextField.tsx:179]: Typically any new feature we add should not affect dom in any way so integration tests would not break. 
In this case we can get away with introducing another div component but we should not add empty helper text.
```suggestion
{props.helperText &&
      <HelperText>
        <HelperTextItem>{props.helperText}</HelperTextItem>
      </HelperText>}
```

**Comment by wtrocki** [comment]
@ljmotta I hope that this change would not affect your implementation in any way. We are only adding new hints to pages.

**Comment by wtrocki** [comment]
LGTM - although I would like us to add helperText to example schemas/add some micro test coverage if possible. 
If you feel that this is not needed let me know. Will merge and release it for ya

**Review by wtrocki — APPROVED** [review]

**Comment by jomarko** [comment]
@wtrocki working on tests

File: src/TextField.tsx [hunk 2]
```diff
@@ -170,6 +174,11 @@ const Text = (props: TextFieldProps) => {
         value={props.value ?? ''}
         {...filterDOMProps(props)}
       />
+{props.helperText &&
+      <HelperText>
+        <HelperTextItem>{props.helperText}</HelperTextItem>
+      </HelperText>}
```
Comment (jomarko) [inline src/TextField.tsx:180]: Removing this code,, as there is osme magic generating automatically help text as shown bellow. It was enough to add `helperText` to `Props`


```
      <TextField required={false} name="x" helperText="y">
        <Text required={false} name="x" helperText="y" id="uniforms-0000-000f" disabled={false} label="" placeholder="" showInlineError={false} changed={false} error={{...}} errorMessage="" field={{...}} fieldType={[Function: String]} fields={{...}} onChange={[Function (anonymous)]} value={[undefined]}>
          <FormGroup fieldId="uniforms-0000-000f" label="" isRequired={false} validated="default" type={[undefined]} helperText="y" helperTextInvalid="">
            <div type={[undefined]} className="pf-c-form__group">
              <div className="pf-c-form__group-control">
                <TextInput name="x" isDisabled={false} validated="default" onChange={[Function: onChange]} placeholder="" type="text" value="" required={false} helperText="y" id="uniforms-0000-000f">
                  <TextInputBase name="x" isDisabled={false} validated="default" onChange={[Function: onChange]} placeholder="" type="text" value="" required={false} helperText="y" id="uniforms-0000-000f" innerRef={{...}} aria-label={{...}} className="" isRequired={false} isReadOnly={false} isLeftTruncated={false}>
                    <input name="x" placeholder="" required={false} helperText="y" id="uniforms-0000-000f" aria-label={{...}} onFocus={[Function (anonymous)]} onBlur={[Function (anonymous)]} className="pf-c-form-control" onChange={[Function (anonymous)]} type="text" value="" aria-invalid={false} disabled={false} readOnly={false} />
                  </TextInputBase>
                </TextInput>
                <div className="pf-c-form__helper-text" id="uniforms-0000-000f-helper" aria-live="polite">
                  y
                </div>
              </div>
            </div>
          </FormGroup>
        </Text>
      </TextField>

```

**Comment by jomarko** [comment]
@wtrocki please have a look when you have a moment

**Review by wtrocki — APPROVED** [review]

**Pull request merged**